src/main/kotlin/no/skatteetaten/aurora/boober/feature/AbstractDeployFeature.kt

```diff
@@ -83,7 +83,7 @@ val AuroraDeploymentSpec.namespace
             else -> "$affiliation-$envName"
         }
     }
-val AuroraDeploymentSpec.releaseTo: String? get() = this.getOrNull<String>("releaseTo")?.takeUnless { it.isEmpty() }
+val AuroraDeploymentSpec.releaseTo: String? get() = if (this.type == TemplateType.deploy) { this.getOrNull<String>("releaseTo")?.takeUnless { it.isEmpty() } } else { null }
```

Comment:
Etter vi diskuterte denne så kom jeg på at vi har job og cronjob som kanskje kan være aktuelt å ha releaseTo på?

Reply:
Vet vi om det er noen i dag som bruker releaseTo på job og cronjob? jeg fant ikke noen når jeg søkte i bitbucket.
Jeg kan sjekke om de fungerer med releaseTo gjennom deploy application testen, hvis det ikke tryner så tar jeg de med i if-sjekken. Hvis det tryner så kan vi ta det som en vurderingsoppgave.

Reply:
Vi kan begrense det til Deploy og CronJob for nå.